evap/static/ts/eslint.config.js

```diff
@@ -0,0 +1,54 @@
+// @ts-check
+
+import eslint from "@eslint/js";
+import tseslint from "typescript-eslint";
+
+export default tseslint.config(
+    eslint.configs.recommended,
+    ...tseslint.configs.strictTypeChecked,
+    ...tseslint.configs.stylisticTypeChecked,
+    {
+        ignores: ["rendered/", "eslint.config.js"],
+    },
+    {
+        languageOptions: {
+            ecmaVersion: 5,
```

Comment:
Don't we use features from a newer version already? Does anything break when using a more recent version?

Reply:
actually, the default is "latest" and I don't think there is a reason why I set this in the first place. I can also set it to 2019 as that is our es target specified in `tsconfig.json`?